src/Sylius/Bundle/CoreBundle/Fixture/Factory/ChannelExampleFactory.php

```diff
@@ -102,6 +104,7 @@ public function create(array $options = [])
         $channel->setHostname($options['hostname']);
         $channel->setEnabled($options['enabled']);
         $channel->setColor($options['color']);
+        $channel->setDefaultTaxCalculationStrategy($options['default_tax_calculation_strategy']);
```

Comment:
It is not really "Default tax calculation strategy" it is just tax calculation strategy. ;)